nil pointer dereference in reporter

<!-- If you need to report a security issue with Knative, send an email to knative-security@googlegroups.com. -->
## In what area(s)?

<!-- Remove the '> ' to select -->

<!--
Other classifications:
> /kind good-first-issue
> /kind process
> /kind spec
> /kind proposal
-->

## What version of Knative?

<!-- Delete all but your choice -->

0.5.2

## Expected Behavior

<!-- Briefly describe what you expect to happen -->
no nil pointer dereference.

## Actual Behavior

<!-- Briefly describe what is actually happening -->
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x1165e71]

goroutine 40 [running]:
github.com/knative/serving/pkg/queue.(*PrometheusStatsReporter).Report(0xc0001abc20, 0x0, 0x0, 0x0)
	/go/src/github.com/knative/serving/pkg/queue/prometheus_stats_reporter.go:117 +0xb1
main.reportStats(0xc000090900)
	/go/src/github.com/knative/serving/cmd/queue/main.go:126 +0x70
```

## Steps to Reproduce the Problem

<!-- How can a maintainer reproduce this issue (be detailed) -->
I have two clusters, both knative serving version are 0.5.2.
First one install knative  by hand, and another enable CloudRun on GKE,
and both have the same error which a little bit annoying.
It seems that *PrometheusStatsReporter may have nil value. 
https://github.com/knative/serving/blob/release-0.5/pkg/queue/prometheus_stats_reporter.go#L117

--- a/cmd/queue/main.go
+++ b/cmd/queue/main.go
@@ -128,11 +128,7 @@ func initEnv() {
 }
 
 func reportStats(statChan chan *autoscaler.Stat) {
-	for {
-		s, ok := <-statChan
-		if !ok {
-			break
-		}
+	for s := range statChan {
 		if err := promStatReporter.Report(s); err != nil {
 			logger.Errorw("Error while sending stat", zap.Error(err))
 		}

--- a/cmd/queue/main.go
+++ b/cmd/queue/main.go
@@ -129,7 +129,10 @@ func initEnv() {
 
 func reportStats(statChan chan *autoscaler.Stat) {
 	for {
-		s := <-statChan
+		s, ok := <-statChan
+		if !ok {
+			break
+		}
 		if err := promStatReporter.Report(s); err != nil {
 			logger.Errorw("Error while sending stat", zap.Error(err))
 		}

